### PR TITLE
codeintel: Revive braindot

### LINKED ISSUE #48358: Strip braindot menu to show "stop light" health of current directory tree

### PR DESCRIPTION
**Pull request by efritz**
Fixes #48358 by reviving the braindot badge to mirror the status of the repo's code graph dashboard page.

Screenshot 2023-03-10 at 12 18 24 PM
Screenshot 2023-03-10 at 12 17 30 PM
Screenshot 2023-03-10 at 12 17 22 PM

image

## Test plan

Tested locally.

## App preview:

- [Web](https://sg-web-ef-48358.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| -0.56% (-17.44 kb) 🔽 | -0.02% (-3.40 kb) | 0.12% (+14.04 kb) 🔺 | 0.27% (+2) 🔺 |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/ef/48358/compare-report.html;tab=live_object) for a full comparison between the commits 08cddc0 and 46cad76 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

**Review by mrnugget — APPROVED** [review]
Let‘s go! (Haven’t looked at code)

File: client/web/src/enterprise/codeintel/dashboard/components/BrainDot.module.scss [hunk 1]
```diff
@@ -19,10 +19,14 @@
...
 }
 
-.braindot-error::after {
-    background-color: var(--red);
+.braindot-success::after {
+    background-color: var(--green);
+}
+
+.braindot-warning::after {
+    background-color: var(--yellow);
```
Comment (umpox) [inline client/web/src/enterprise/codeintel/dashboard/components/BrainDot.module.scss:27]:
```suggestion
.braindot-success::after {
    background-color: var(--success);
}

.braindot-warning::after {
    background-color: var(--warning);
```

We should use semantic variables here so they stay in sync with our other indicators (no change in colors right now)

File: client/web/src/enterprise/codeintel/dashboard/components/BrainDot.module.scss [hunk 2]
```diff
@@ -19,10 +19,14 @@
...
 }
 
-.braindot-attention::after {
-    background-color: var(--purple);
+.braindot-danger::after {
+    background-color: var(--red);
```
Comment (umpox) [inline client/web/src/enterprise/codeintel/dashboard/components/BrainDot.module.scss:31]:
```suggestion
    background-color: var(--danger);
```

**Review by umpox — APPROVED** [review]
👍

**Pull request merged**